### PR TITLE
SG-20889 Remove dead code related to UI tests on Azure

### PR DESCRIPTION
**Pull request by carlos-villavicencio-adsk**
We plan to migrate these tests to Jenkins.

Tested here: https://github.com/shotgunsoftware/tk-core/pull/980

### CODE REVIEW

File: build-pipeline.yml [hunk 1]
```diff
@@ -32,18 +32,14 @@ parameters:
     type: string
     default: master
   # Allows to skip test execution.
-  # This parameter is deprecated. Please use has_unit_tests and has_ui_tests.
```
Comment (julien-lang) [inline build-pipeline.yml:35]: If that's that deprecated. Can we remove that too?

**Review by julien-lang — COMMENTED** [review]
I am not sure about that. I see that the following components are setting `has_ui_tests: true`:

- tk-multi-shotgunpanel
- tk-multi-about
- tk-multi-publish2
- tk-multi-loader2
- tk-multi-workfiles2
- tk-multi-pythonconsole

File: build-pipeline.yml (hunk 1 above)
Comment (carlos-villavicencio-adsk) [reply build-pipeline.yml:35]: Unfortunately, it's still in use by many tookit components.

**Comment by carlos-villavicencio-adsk** [comment]
> I am not sure about that. I see that the following components are setting `has_ui_tests: true`:
> 
> * tk-multi-shotgunpanel
> * tk-multi-about
> * tk-multi-publish2
> * tk-multi-loader2
> * tk-multi-workfiles2
> * tk-multi-pythonconsole

@julien-lang true! However, they're always skipped because we cannot possibly run these tests on Azure. Removing the flag here won't break repos that are using it on `true`.

image

**Comment by julien-lang** [comment]
> @julien-lang true! However, they're always skipped because we cannot possibly run these tests on Azure. Removing the flag here won't break repos that are using it on `true`.

Ok then we might want to do some more cleanup/refactoring then. And have a dedicated ticket for that.

**Review by julien-lang — APPROVED** [review]

**Pull request merged**